Packs/SentryWire/Integrations/SentryWire/SentryWire.yml

```diff
@@ -0,0 +1,454 @@
+category: Network Security
+commonfields:
+  id: SentryWire
+  version: -1
+configuration:
+- display: Unit Address
+  name: unitaddress
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+description: Utilize SentryWire's search capabilities. For more information please refer to SentryWire documentation
+display: SentryWire
+name: SentryWire
+script:
+  commands:
+  - arguments:
+    - default: false
+      isArray: false
+      name: search_name
+      required: true
+      secret: false
+    - default: false
+      isArray: false
+      name: search_filter
+      required: true
+      secret: false
+    - default: false
+      isArray: false
+      name: begin_time
+      required: true
+      secret: false
+    - default: false
+      isArray: false
+      name: end_time
+      required: true
+      secret: false
+    - default: false
+      isArray: false
+      name: max_packets
+      required: false
+      secret: false
+    - default: false
+      isArray: false
+      name: fmgroups
+      required: false
+      secret: false
+    - default: false
+      isArray: false
+      name: generate_links
+      required: false
+      secret: false
+    deprecated: false
+    description: Create a search on SentryWire unit
+    execution: false
+    name: sentrywire-create-search
+  - arguments:
+    - default: false
+      isArray: false
+      name: SearchID

```

Comment:
Argument names should follow the pattern **search_id**, not **SearchID**. This applies to all arguments in all commands.